.github/CODEOWNERS

```
@@ -0,0 +1,20 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence, @faker-js/maintainers will 
+# be requested for review when someone opens a pull request.
+
+*  @faker-js/maintainers
+
+# ================================================
+#  Docs owners ...
+# ================================================
+
+/docs/                                                          @faker-js/docs-maintainers
+netlify.toml                                                    @faker-js/docs-maintainers 
```

The team is called `maintainers-docs` instead of `docs-maintainers`

```suggestion
/docs/                                                          @faker-js/maintainers-docs
netlify.toml                                                    @faker-js/maintainers-docs
```